azurerm/internal/services/compute/tests/resource_arm_linux_virtual_machine_scale_set_disk_os_test.go

```diff
@@ -287,6 +312,166 @@ resource "azurerm_linux_virtual_machine_scale_set" "test" {
 `, template, data.RandomInteger, diskSize)
 }
 
+func testAccAzureRMLinuxVirtualMachineScaleSet_disksOSDisk_diskEncryptionSetDependencies(data acceptance.TestData) string {
+	// whilst this is in Preview it's only supported in: West Central US, Canada Central, North Europe
+	// TODO: switch back to default location
+	location := "westus2"
+
+	return fmt.Sprintf(`
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestrg-%d"
```

Comment:
```suggestion
  name     = "acctestRG-%d"
```
? 🤔

Reply:
this is out-of-scope for this PR, since this wants to be made to the Linux VMSS/Disk Encryption Set template rather than specific to this test-case